src/nng.c

```diff
@@ -626,7 +626,8 @@ nng_setopt(nng_socket s, const char *name, const void *val, size_t sz)
 }
 
 static int
-nni_socket_getx(nng_socket s, const char *name, void *val, size_t *szp, nni_type t)
+nni_socket_getx(
+    nng_socket s, const char *name, void *val, size_t *szp, nni_type t)
 {
```

Comment:
Interestingly, my clang-format has concluded this should be reformatted... (?)

Reply:
What version of clang-format are you using?  Sometimes new versions change things...

Reply:
Actually mine does too now.